Added IPS Citra and Luma guide (#76)
zaksabeast/PokemonRNGGuides: guides/tools/en/Apply 3DS IPS Patches to Citra.md

# Apply 3DS IPS Patches to Citra

---
## Getting your game's code.bin
---

> 1. Boot your 3DS into GodMode9

Reviewer: ```suggestion
1. Boot your 3DS into GodMode9 by holding `Start` while powering on the 3DS.
```

> 2. Click "SYSNAND SD"

Reviewer: Since you can't click with a 3DS, "choose" in place of "click" sounds better.
```suggestion
2. Choose "SYSNAND SD"
```

> 3. Click "title"

Reviewer: ```suggestion
3. Choose "title"
```

> 4. Click the first half of your game's title ID

Reviewer: ```suggestion
4. Choose the first half of your game's title ID
```

You mention later about UM's update title ID, but here you mention the game's title ID. For applying ips patches would you want the game's code.bin or the game update's code.bin?

> 5. Click the second half of your game's title ID

Reviewer: ```suggestion
5. Choose the second half of your game's title ID
```

> 6. Click the `content` folder

Reviewer: ```suggestion
6. Choose the `content` folder
```

> 7. On the top screen, click on the `.app` file

Reviewer: ```suggestion
7. On the top screen, choose the `.app` file
```

> 8. On the bottom screen, click `NCCH image options...`

Reviewer: ```suggestion
8. On the bottom screen, choose `NCCH image options...`
```

> 9. Click `Extract .code`

Reviewer: ```suggestion
9. Choose `Extract .code`
```

    - Your `.code` file is in `gm9/out/titleID.code`
    - For example, Ultra Moon's update would be `0004000E001B5100.code`

> 10. Get the `.code` file to your computer using an SD reader or FTP

Reviewer: ```suggestion
10. Transfer the `.code` file to your computer using an SD card reader or FTP
```

> Here is an example of what to click for Ultra Moon's update (title ID `0004000E001B5100`):

Reviewer: ```suggestion
Here is an example of what to choose for Ultra Moon's update (title ID `0004000E001B5100`):
```
